File: SEM3_CODE/PHP/BASIC/12_sum_numbers.php
<?php
$limit = (int) readline('Enter the last number: ');  // Read the last number
$total = 0;                                          // Store the running total

for ($number = 1; $number <= $limit; $number++) {    // Visit numbers up to limit
    $total = $total + $number;                       // Add the current number
}

echo 'Sum: ' . $total;                               // Display the total
?>

File: SEM5_CODE/PYTHON/BASIC/12_sum_numbers.py
limit = int(input("Enter the last number: "))  # Read the last number
total = 0                                      # Store the running total

for number in range(1, limit + 1):             # Visit numbers from 1 to limit
    total = total + number                     # Add the current number

print("Sum:", total)                           # Display the total
